client/package.json

```diff
@@ -107,6 +107,7 @@
   },
   "dependencies": {
     "@bugsnag/js": "^7.16.2",
+    "@capacitor/core": "^4.7.1",
```

Comment:
the core versions were inconsistent - I've moved to the root rather than common, since the plugin was using Capacitor, and have left it here. We've used Capacitor in `useNativeClient` and that's used in a number of packages so it makes sense to have it at the root.